README.md

```diff
@@ -127,7 +127,8 @@ you would need to link them in your "pack" or entry file.
 
 ### Development
 
-Webpacker ships with two binstubs: `./bin/webpack` and `./bin/webpack-dev-server`.
+Webpacker ships with two binstubs which are available after installing the gem:
```

Comment:
We should probably refer to this as binaries rather than binstubs, since there are not binstubs anymore.